src/ewf_adapter_renesas_gm01q.h

```diff
@@ -0,0 +1,465 @@
+ * EWF_CONFIG_ADAPTER_RENESAS_GM01Q_USER_CA_CERTIFICATE_ID
+ */
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CA_CERTIFICATE_ID
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_CA_CERTIFICATE_ID                    EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CA_CERTIFICATE_ID
+#endif
+
+/**
+ * @brief Client certificate index number(0-19)
+ * The index in the non volatile memory where the client certificate will be stored
+ *
+ */
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_CERTIFICATE_ID
+#undef EWF_CONFIG_ADAPTER_RENESAS_COMMON_CLIENT_CERTIFICATE_ID
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_CLIENT_CERTIFICATE_ID                EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_CERTIFICATE_ID
+#endif
+
+/**
+ * @brief Client Key index number(0-19)
+ * The index in the non volatile memory where the client key will be stored
+ *
+ */
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_PRIVATE_KEY_ID
+#undef EWF_CONFIG_ADAPTER_RENESAS_COMMON_CLIENT_PRIVATE_KEY_ID
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_CLIENT_PRIVATE_KEY_ID                EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_PRIVATE_KEY_ID
+#endif
+
+/** @brief Trusted Certificate Authority certificate pointer name*/
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_ROOT_CA_CERTIFICATE_BUFFER_NAME
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_ROOT_CA_CERTIFICATE_BUFFER_NAME      EWF_CONFIG_ADAPTER_RENESAS_GM01Q_ROOT_CA_CERTIFICATE_BUFFER_NAME
+#endif
+
+/** @brief User Certificate Authority certificate pointing buffer name */
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_USER_CA_CERTIFICATE_BUFFER_NAME
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_USER_CA_CERTIFICATE_BUFFER_NAME      EWF_CONFIG_ADAPTER_RENESAS_GM01Q_USER_CA_CERTIFICATE_BUFFER_NAME
+#endif
+
+/** @brief Client certificate pointing buffer name */
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_CERTIFICATE_BUFFER_NAME
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_CLIENT_CERTIFICATE_BUFFER_NAME       EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_CERTIFICATE_BUFFER_NAME
+#endif
+
+/** @brief Client private key filename  */
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_PRIVATE_KEY_BUFFER_NAME
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_CLIENT_PRIVATE_KEY_BUFFER_NAME       EWF_CONFIG_ADAPTER_RENESAS_GM01Q_CLIENT_PRIVATE_KEY_BUFFER_NAME
+#endif
+
+/**
+ * @brief Security Profile ID (0-6)
+ * The id number for the security profile configuration
+ *
+ */
+#ifdef EWF_CONFIG_ADAPTER_RENESAS_GM01Q_SECURITY_PROFILE_ID
+#undef EWF_CONFIG_ADAPTER_RENESAS_COMMON_SECURITY_PROFILE_ID
+#define EWF_CONFIG_ADAPTER_RENESAS_COMMON_SECURITY_PROFILE_ID                  EWF_CONFIG_ADAPTER_RENESAS_GM01Q_SECURITY_PROFILE_ID
+#endif
+
+/************************************************************************//**
+ * @} *** group_configuration_adapter_renesas_gm01q
+ ****************************************************************************/
+
+/* Include the Quectel common header here, to take the above configuration macros into account */
```

Comment:
There are a few instances of "Quectel" in these files that need replaced.

Reply:
changed